requirements.txt

```diff
@@ -4,6 +4,7 @@ scipy>=0.18
 scikit-learn>=0.18
 numpy>=1.11.0
 cycler>=0.10.0
+pytest>=3.0.7
```

Comment:
We are a bit wary about adding dependencies to Yellowbrick; we have had a lot of trouble with dependency management for different versions as well as building other tools like Conda packages. At the very least, this should be an optional dependency for developers and placed into `tests/requirements.txt` rather than the global  requirements. However, I don't believe that pytest is required here, and your use case can be easily replaced with unittest; see below.